l10n_ar_withholding_ux/views/account_tax_view.xml

```diff
@@ -47,6 +47,7 @@
                             </div>
                             <field name="withholding_non_taxable_minimum"/>
                             <field name="withholding_non_taxable_amount"/>
+                            <field name="ratio" invisible="country_code != 'AR'"/>
```

Comment:
<label for="ratio" invisible="country_code != 'AR'/>
                <div invisible="country_code != 'AR'>
                    <field name="ratio" class="oe_inline"/>
                    %
                </div>